modules/games/definitions/jesonmor/performance.ts

```diff
@@ -0,0 +1,9 @@
+import { JesonmorDefinition } from "./_types";
+
+const jesonmorPerformance: JesonmorDefinition["performance"] = {
+  canAlwaysEnd: {},
```

Comment:
Not that it matters in this game, but if we select where to go, we can always end:

```suggestion
  canAlwaysEnd: {
    selectmovetarget: true
  },
```

This means the engine doesn't actually have to perform the `move` command when trying to prune dead branches

Reply:
Ah okay!